app/controllers/cockpit_controller.rb

```diff
@@ -5,15 +5,11 @@ def host_ssh_params
     render :json => SSHExecutionProvider.ssh_params(@host)
   end
 
-  def cockpit_url_for_host(host)
```

Comment:
Since we're using this at two places, moved the code the `SSHExecutionProvider`, as I could not think of better place, and adding separate class seemed like an overkill